qnexus/client/hugr.py

```diff
@@ -288,14 +290,6 @@ def _fetch_hugr_package(handle: HUGRRef) -> Package:
         raise qnx_exc.ResourceFetchFailed(message=res.text, status_code=res.status_code)
 
     hugr_base_64_bytes = res.json()["data"]["attributes"]["contents"]
```

Comment:
I suggest renaming this variable and giving a type hint so `mypy` can make sure of things.  `res.json()["data"]["attributes"]["contents"]` is a `str`.  I've checked.
```suggestion
    contents: str = res.json()["data"]["attributes"]["contents"]
```

(applying this suggestion alone will be insufficient; the variable is used later)